worker/wallet.js

```diff
@@ -305,35 +303,18 @@ export async function checkWithdrawal ({ data: { hash, withdrawal, invoice }, bo
     ], { models })
     if (code === 0) {
       notifyWithdrawal(dbWdrwl.userId, wdrwl)
-      if (dbWdrwl.wallet) {
-        // this was an autowithdrawal
-        const message = `autowithdrawal of ${
-          numWithUnits(msatsToSats(paid), { abbreviate: false })} with ${
-          numWithUnits(msatsToSats(fee), { abbreviate: false })} as fee`
-        await addWalletLog({ wallet: dbWdrwl.wallet, level: 'SUCCESS', message }, { models })
```

Comment:
we don't need this because we log success/errors using `.then` and `.catch` on `payViaPaymentRequest`

Reply:
I changed this to log in `paidActionForwarded` etc.